release.env

```diff
@@ -0,0 +1,3 @@
+RELEASE_DIR="<Release dir on server>"
+RELEASE_WEB="https://raw.githubusercontent.com/DOMjudge/domjudge-scripts/master/make_release.sh"
```

Comment:
I would rather run this from a local checkout of `domjudge-scripts.git`. We already have and use that at `domjudge@domjudge.org:domjudge-scripts`.